server/notification-providers/slack.js

```diff
@@ -58,26 +139,7 @@ class Slack extends NotificationProvider {
                 "attachments": [
                     {
                         "color": (heartbeatJSON["status"] === UP) ? "#2eb886" : "#e01e5a",
-                        "blocks": [
-                            {
-                                "type": "header",
-                                "text": {
-                                    "type": "plain_text",
-                                    "text": textMsg,
-                                },
-                            },
-                            {
-                                "type": "section",
-                                "fields": [{
-                                    "type": "mrkdwn",
-                                    "text": "*Message*\n" + msg,
-                                },
-                                {
-                                    "type": "mrkdwn",
-                                    "text": `*Time (${heartbeatJSON["timezone"]})*\n${heartbeatJSON["localDateTime"]}`,
-                                }],
-                            }
-                        ],
+                        "blocks": blocks,
```

Comment:
nit: I think inlining this is better, as blocks is not used elesewhere and this is quite compact this way

```suggestion
                        "blocks": await Slack.buildBlocks(monitorJSON, heartbeatJSON, textMsg, msg),
```